src/zarr_benchmarks/fetch_datasets.py

```diff
@@ -21,6 +21,6 @@ def get_image() -> npt.NDArray:
     heart_image_path = ZENODO.path / f"{heart_image}.zip.unzip" / heart_image
```

Comment:
Potentially you might want to fix the plots in this PR too? Currently `zarr_spec` v2/v3 points will be mixed together as there is no filtering. You could modify `create_plots` to retain the `zarr_spec` column in `prepare_benchmarks_dataframe` then add a filter for zarr spec v2? Then plots for v3 could be added in another PR.